libfwupdplugin/fu-device.c

```diff
@@ -230,6 +230,8 @@ fu_device_internal_flag_to_string(FuDeviceInternalFlags flag)
 		return "use-parent-for-battery";
 	if (flag == FU_DEVICE_INTERNAL_FLAG_USE_PROXY_FALLBACK)
 		return "use-proxy-fallback";
+	if (flag == FWUPD_DEVICE_FLAG_NO_AUTO_REMOVE)
```

Comment:
Since moved to this file needs "internal" in flag name

Reply:
my bad fixed it